openid-4-verifiable-presentations-1_0.md

```diff
@@ -1786,6 +1786,16 @@ The OpenID Foundation provides tools that can be used to confirm that an impleme
 
 https://openid.net/certification/conformance-testing-for-openid-for-verifiable-presentations/
 
+# Privacy Considerations {#privacy_consideratins}
+
+Many privacy considerations are specific to the credential format and associated proof type used in any particular presentation.
```

Comment:
nit: "used in a particular presentation"

Reply:
```suggestion
Many privacy considerations are specific to the credential format and associated proof type used in a particular presentation.
```